allensdk/brain_observatory/ecephys/ecephys_project_cache.py

```diff
@@ -92,7 +92,11 @@ def reader(path):
 
     def _get_probes(self):
         path = self.get_cache_path(None, self.PROBES_KEY)
-        return call_caching(self.fetch_api.get_probes, path, strategy='lazy', **csv_io)
+        probes = call_caching(self.fetch_api.get_probes, path, strategy='lazy', **csv_io)
+        # Divide the lfp sampling by the subsampling factor for clearer presentation
+        probes["lfp_sampling_rate"] = (
+            probes["lfp_sampling_rate"] / probes["lfp_temporal_subsampling_factor"])
+        return probes
```

Comment:
could you remove the subsampling factor after application?

Reply:
Added to the list of suppressed columns